### ISSUE TITLE
Multiple "Use of undeclared Var" warnings when using shadow-cljs

### ISSUE THREAD

**Issue by p-himik** [Day8/re-frame-10x#173]
`shadow-cljs` is a bit more strict, so using `some.thing/foo` without `(:require [some.thing])` produces `Use of undeclared Var some.thing/foo` warnings.

The cases in `re-frame-10x` that were reported by `shadow-cljs`:
```
day8/re_frame_10x/view/parts.cljs:26:28
day8/re_frame_10x/view/parts.cljs:28:34
day8/re_frame_10x/view/parts.cljs:30:34
```
There may be other cases - I'm not sure.

**Comment by danielcompton** [comment]
This should be fixed in 0.2.1 and above, can you confirm whether you are getting these warnings with 0.3.0?

Relates to #164.

**Comment by p-himik** [comment]
Works perfectly, thank you!
